docs/integrations/changelog.md

```diff
@@ -5,82 +5,89 @@ title: Change Log
 
 A change log file helps to keep track on the changes made for a specific content entity like an integration or a playbook.
 
-## Naming
-Under the same path of the changed content entity add a new file:
- - In case of a package, file name is `CHANGELOG.md`.
- - otherwise, file name has the same base name with the suffix of `_CHANGELOG.md`.
+To generate a change log, run the following command provided by the `demisto-sdk`:
 
-For example,
-- If the changed file is `Integrations/Alexa/Alexa.py` then create `Integrations/Alexa/CHANGELOG.md` with the release notes inside.
-- If the changed file is `Integrations/integration-jira.yml` then create `Integrations/integration-jira_CHANGELOG.md` with the release notes inside.
-- If the changed file is `Playbooks/playbook-Phishing.yml` then create `Playbooks/playbook-Phishing_CHANGELOG.md` with the release notes inside.
+```bash
+demisto-sdk update-release-notes -p [Changed Pack Name] -u [major|minor|revision]
```

Comment:
where do we have the flow of overwriting the existing new RN on the branch in order to update it? (without bumping the version)

Reply:
Just added.